src/dolphin/os/OSInterrupt.h

```diff
@@ -44,6 +44,17 @@ typedef enum {
     OS_INTR_MAX
 } OSInterruptType;
 
+typedef u32 OSInterruptMask;
+
+#define OS_INTRMASK_MEM_0 (0x80000000U >> OS_INTR_MEM_0)
+#define OS_INTRMASK_MEM_1 (0x80000000U >> OS_INTR_MEM_1)
+#define OS_INTRMASK_MEM_2 (0x80000000U >> OS_INTR_MEM_2)
+#define OS_INTRMASK_MEM_3 (0x80000000U >> OS_INTR_MEM_3)
+#define OS_INTRMASK_MEM_ADDRESS (0x80000000U >> OS_INTR_MEM_ADDRESS)
+#define OS_INTRMASK_DSP_AI (0x80000000U >> OS_INTR_DSP_AI)
+#define OS_INTRMASK_DSP_ARAM (0x80000000U >> OS_INTR_DSP_ARAM)
+#define OS_INTRMASK_DSP_DSP (0x80000000U >> OS_INTR_DSP_DSP)
+#define OS_INTRMASK_AI_AI (0x80000000U >> OS_INTR_AI_AI)
```

Comment:
It's whatever for this PR but wouldn't it make sense to `#define BASE_VIRT_ADDR 0x80000000U` or something?

Reply:
This isn't an address, it's a mask value.